runtime/ts/description.ts

```diff
@@ -123,20 +123,16 @@ export class DescriptionFormatter {
     this.particleDescriptions = [];
 
     // Combine all particles from direct and inner arcs.
-    const innerParticlesByName = {};
+    const innerParticles = [];
     description.arc.recipes.forEach(recipe => {
       const innerArcs = [...recipe.innerArcs.values()];
       innerArcs.forEach(innerArc => {
         innerArc.recipes.forEach(innerRecipe => {
-          innerRecipe.particles.forEach(innerParticle => {
-            if (!innerParticlesByName[innerParticle.name]) {
-              innerParticlesByName[innerParticle.name] = innerParticle;
-            }
-          });
+          innerParticles.push(...innerRecipe.particles);
         });
       });
     });
-    const allParticles = description.arc.activeRecipe.particles.concat(Object.values(innerParticlesByName));
+    const allParticles = description.arc.activeRecipe.particles.concat(innerParticles);
```

Comment:
Why do you want to change it now? This be adding multiple of the same hosted particle, and I see how the current way is not be exactly right, but i did it this way because for TV shows for example, the clean way gets everything either too slow or out of memory IIRC.

Reply:
The problem with fetching it by name will appear not just in weird cases, but any recipe that has 2 of the same particle (not necessarily hosted, just regular particle), so I think this hack could only work this way and if we want to truly resolve it, we'll need to do the cleanup. 

Alternatively, maybe particle needs to have 2 types of description - one for when it is standalone, other for when it is multiplexed?
Actually, say, the `PlayRecordRenderer` - if the suggestion is to just show the record, the description should be `Show "No One Knows"`. But if it is multiplexed, this description is of no use, we really only need the Multiplexer particle description.

Reply:
I haven't thought about descriptions for multiplexed recipes/particles, there's definitely some interesting problem there - but I'll leave that for now.

The take-away for me here is that there's another reason to look into cleaning up stale particles from the transformation.